examples/demo.yaml

```diff
@@ -8,6 +8,7 @@ apiVersion: v1
 kind: PersistentVolume
 metadata:
   name: demo
+  namespace: default
```

Comment:
This is a fairly debatable change, it's good for us when testing locally, but not helpful for users who want to use this to demo the controller. Thoughts?

Reply:
I would definitely keep that change if that's suitable for testing